### PR TITLE
rdbc-663

### PR DESCRIPTION
**Pull request by alxsabo**
https://issues.hibernatingrhinos.com/issue/RDBC-663

### CODE REVIEW

File: src/Documents/Session/Loaders/QueryIncludeBuilder.php
Comment (ml054) [inline src/Documents/Session/Loaders/QueryIncludeBuilder.php:85]: what about that?

File: src/Documents/Session/TimeSeries/TimeSeriesValuesHelper.php
Comment (ml054) [inline src/Documents/Session/TimeSeries/TimeSeriesValuesHelper.php]: cache? ;-)

**Comment by ml054** [comment]
@alxsabo Could you check failing test + fix (if needed my 2 remarks) before merge?

File: src/Documents/Session/Loaders/QueryIncludeBuilder.php
Comment (alxsabo) [reply src/Documents/Session/Loaders/QueryIncludeBuilder.php:85]: none of the test required it yet, it just have to be uncommented. base method is implemented

File: src/Documents/Session/TimeSeries/TimeSeriesValuesHelper.php
Comment (alxsabo) [reply src/Documents/Session/TimeSeries/TimeSeriesValuesHelper.php]: cash is better :D

File: src/Documents/Session/Loaders/QueryIncludeBuilder.php
Comment (ml054) [reply src/Documents/Session/Loaders/QueryIncludeBuilder.php:85]: ok sure. :)

**Review by ml054 — APPROVED** [review]

**Comment by ml054** [comment]
Great let's wait for GH actions and we can merge. :)

**Pull request merged**
